CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ Pull requests are the best way to propose changes to the codebase.
 
 1. Fork the repo and create your branch from `master`.
 2. If you've changed something, update the documentation.
-3. Make sure your code lints (using pre-commit).
+3. Make sure your code lints (using `pre-commit`).
```

Comment:
Shouldn't it say  "Make sure your code lints passes"

Reply:
Updated this.